### ISSUE TITLE
Should AD be communicated in the Join message, to allow consensus to form around it?

### ISSUE THREAD

**Issue by ftrader** [tyler-smith/snowglobe#3]
The spec suggests that 

>  in the long term participants should recognize a particular Acceptance Depth

But the protocol does not seem to accomodate an exchange of that parameter to allow some kind of consensus to form on it.

An immediate question I have would be: 

Can complications or problems for the protocol arise if a node is using some value of AD, but peers joining to it all use different values?

**Comment by tyler-smith** [comment]
Yeah, I'm still researching what suitable values might be and how
implementations might want to set them. There's a trade off here that could
effect operators as well which is that a longer AD means a heavier reorg if
it ever triggers.

It's also not yet clear to me if non synchronized values would necessarily
lead to consensus failures or if it's possible to have a scheme such that
differing values just affect the time a given node is willing to try
holding onto the Ava consensus.

This is definitely one of the areas that needs to most work in terms of
research and modeling imo.

**Comment by imaginaryusername** [comment]
Depends on how robust the sybil resistance is, it might be more reliable to have a flat value for AD. Note that the sybil resistance in Avalanche/Snowglobe can be weaker than, say, NC due to only needing to vote things "up" or "down", but trying to converge on a number (like in AD) might be messier.

**Comment by ftrader** [comment]
I was thinking more along the lines of "this peer trying to join me doesn't have the same AD value, therefore I flat out reject him".

**Comment by tyler-smith** [comment]
Sounds like a good idea.  I'll have to think about that some more.
